packages/commonwealth/client/scripts/views/components/header/login_selector.tsx

```diff
@@ -197,6 +199,22 @@ export const LoginSelectorMenuRight = ({
     localStorage.getItem('dark-mode-state') === 'on'
   );
 
+  const resetWalletConnectSession = async () => {
+    /**
+     * Imp to reset wc session on logout as subsequent login attempts fail
+     */
+    const chainbase = app.chain?.meta?.base;
+    const wallets = app.wallets.availableWallets(chainbase);
```

Comment:
We should note that behavior here, if chainbase is null, is to return all available wallets. Please test against the landing page (i.e. one where chainbase is null) to confirm that the behavior works as expected.

Reply:
Are we able to write a test here, rather than what I take to be implied here (CA)

Reply:
This is working for me.

Reply:
These code paths should definitely be unit-testable, at least in the wallet-connect case. The other cases too, but they will require mocking values on the window object.